src/md4c.h

```diff
@@ -297,6 +299,12 @@ typedef struct MD_SPAN_WIKILINK {
     MD_ATTRIBUTE target;
 } MD_SPAN_WIKILINK_DETAIL;
 
+/* Detailed info for MD_SPAN_MENTION. */
+typedef struct MD_SPAN_MENTION {
+    unsigned char size; 
+    MD_CHAR* text;
```

Comment:
I understand it's supposed to contain only a verbatim text (the username) with no possible nested formatting options, yet I wonder whether rather using `MD_ATTRIBUTE target` (and initializing it to a single `substr_type` of type `MD_TEXT_NORMAL`) would not be better, also for the sake of consistency with other detail structures.